<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZWebViewBrowser</class>
 <widget class="QWidget" name="ZWebViewBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZWebViewBrowser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ZWebViewBrowser{background-color: rgb(91, 168, 239);}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="name_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 18px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>云财税</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="miniPushButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/mini.png);}
        QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/mini_click.png);}
        QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/mini_click.png);};
      </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>4</width>
       <height>14</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="maxPushButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/max.png);}
        QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/max_click.png);}
        QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/max_click.png);};</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>14</width>
       <height>14</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="closePushButton">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/close.png);}
        QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/close_click.png);}
        QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/close_click.png);};</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 235);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>13</number>
      </property>
      <property name="leftMargin">
       <number>13</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="backPushButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
           QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/left.png);}
           QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/left_click.png);}
           QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/left_click.png);};
         </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>8</width>
          <height>14</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(225, 225, 225);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forwardPushButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
           QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/right.png);}
           QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/right_click.png);}
           QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/right_click.png);};</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(225, 225, 225);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="flushPushButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
           QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/repeat.png);}
           QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/repeat_click.png);}
           QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/repeat_click.png);};
         </string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>414</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
